### DISCUSSION TITLE
Q:在浏览器访问评论管理页面打不开,提示GET https://unpkg.com/@waline/admin net::ERR_TIMED_OUT || Q: The comment management page cannot be opened in the browser, prompting GET https://unpkg.com/@waline/admin net::ERR_TIMED_OUT

### DISCUSSION THREAD

**Discussion by gglss** [walinejs/waline#2232] [Q&A]
### Describe the bug | 问题描述

配置好waline评论后访问管理页面，访问界面为空
![截屏2024-01-04 19 34 37](https://github.com/walinejs/waline/assets/81339751/e2463097-cebd-45e4-9a23-e8477c0ed16b)

### Website URL | 问题网站

https://waline.gglss.top/ui

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

After configuring waline comments, access the management page. The access interface is empty.
![Screenshot 2024-01-04 19 34 37](https://github.com/walinejs/waline/assets/81339751/e2463097-cebd-45e4-9a23-e8477c0ed16b)

### Website URL | Problem website

https://waline.gglss.top/ui

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
same as #2231
